modules/caddyhttp/encode/gzip/gzip.go

```diff
@@ -81,6 +81,9 @@ func (g Gzip) Validate() error {
 // used in the Accept-Encoding request headers.
 func (Gzip) AcceptEncoding() string { return "gzip" }
 
+// Suffix returns the filename suffix of precompressed files.
+func (Gzip) Suffix() string { return ".gz" }
```

Comment:
Is this still needed since there's the `GzipPrecompressed` type?

Reply:
I moved the implementation from `GzipPrecompressed` to `Gzip` because of [this review](https://github.com/caddyserver/caddy/pull/4045#discussion_r585891879)

But I can move it back to `GzipPrecompressed` as it makes more sense there. Same for `ZstdPrecompressed`.
You tell me how you want it :)